ardupilot/source/index.rst

```diff
@@ -1,13 +1,14 @@
 .. _home:
 
+=========
+ArduPilot
```

Comment:
dont see the usefullness of this title...its obvious we are talking about ArduPilot

Reply:
Since that was added to the other pages, it just made them more symmetric. I could go either way on it. What do you think?

Reply:
I think it can be removed